articles/quickstart/webapp/laravel/01-login.md

```diff
@@ -148,7 +134,7 @@ Route::get( '/auth0/callback', '\Auth0\Login\Auth0Controller@callback' )->name(
 
 If you load this callback URL now, you should be immediately redirected back to the homepage rather than getting a 404 error. This tells us that the route is setup and being handled. 
 
-Now we need to add this URL to the **Allowed Callback URLs** field in the Application settings screen for the Application used with this app. Add the complete URL, like `https://myapp.com/auth0/callback`.
+Now we need to add this URL to the **Allowed Callback URLs** field in the Application settings screen for the Application used with this app. This should be your Laravel app's `APP_URL` followed by `/auth0/callback`. If you're using one of our samples, this was already configured at the beginning of this tutorial.
```

Comment:
> If you're using one of our samples...

Does this mean if the user downloaded the sample from the Quickstart? If so maybe be specific so as not to confuse developers thinking if they just cloned the sample instead downloading it from the Quickstart.